whois-api/src/main/java/net/ripe/db/whois/api/rdap/RdapObjectMapper.java

```diff
@@ -215,7 +212,15 @@ private RdapObject getRdapObject(final String requestUrl,
             rdapResponse.getRemarks().add(createRemark(rpslObject));
         }
 
-        rdapResponse.getEvents().add(createEvent(lastChangedTimestamp));
+        if (rpslObject.containsAttribute(AttributeType.CREATED)){
```

Comment:
These attributes *must* always appear in an RPSL object (they are mandatory), i.e. remove the 'if' check

Reply:
Yeah, I know. However: 
Each response that includes an RDAP object as the topmost object SHOULD include a "registration" event and a "last changed" event for that object. This may not be possible for all objects: for example, event data for legacy IP network objects is often missing or unreliable.

This may not be possible for all objects: for example, event data for legacy IP network objects is often missing or unreliable.

Maybe the legacy object does not have this object. I know that it is autogenerated attributes but maybe there are some data in database that do not have this data, could be that possible?

Reply:
Every existing object in the database contains both created and last-modified attributes, so this check should never fail, and if it does it's a bug

Reply:
I'm going to add a check and throw an error if this happens.